### PR TITLE
feat: Mica

### LINKED ISSUE #93: Switch to win32mica
Will do soon!

### PR DESCRIPTION
**Pull request by sumeshir26**
Closes #93

And people say that you cant make nice looking apps in python
image

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -776,16 +777,25 @@ def sizechanged(e):
...
+            ApplyMica(HWND=windll.user32.GetParent(app.winfo_id()), ColorMode=MICAMODE.DARK)
+        else:
+            ApplyMica(HWND=windll.user32.GetParent(app.winfo_id()), ColorMode=MICAMODE.LIGHT)
+    else:
+        from BlurWindow.blurWindow import GlobalBlur
+        if theme == "Dark":
+            GlobalBlur(
+                windll.user32.GetParent(app.winfo_id()),
+                Acrylic=True,
+                Dark=True)
```
Comment (not-nef) [inline main.py:793]: @sumeshir26 You have to add `hexColor="#1c1c1c"`.

**Comment by not-nef** [comment]
Great, now we only need to change the background of the title!

**Comment by sumeshir26** [comment]
@not-nef How would we do that?

File: main.py [hunk 2]
```diff
@@ -776,16 +777,25 @@ def sizechanged(e):
...
+        else:
+            GlobalBlur(
+                windll.user32.GetParent(app.winfo_id()),
+                Acrylic=True,
+                Dark=False,)
```
Comment (not-nef) [inline main.py:798]: Remove this please, we still didnt find a solution for the overexposure.

**Review by not-nef — CHANGES_REQUESTED** [review]

**Comment by not-nef** [comment]
> @not-nef How would we do that?

Idk

File: main.py (hunk 2 above)
Comment (sumeshir26) [reply main.py:798]: Ok

**Comment by sumeshir26** [comment]
It is actuallay a Windows OS bug, and is fixed in the latest Dev build, so we'll have to wait until the fix comes to Beta and Stable channel

**Comment by not-nef** [comment]
Great!! Ill approve the pr when im home.

**Comment by im-coder-lg** [comment]
Wow what in the world? Will it be stable on Windows 7?

**Comment by im-coder-lg** [comment]
> It won’t even apply on win 7, since mica is exclusive to 11

Oh flippin' yeah, testing this!

**Comment by im-coder-lg** [comment]
We have a code error on Linux(Raspbian 64-bit) as well as Windows. I'll post the command logs here in a few moments.

**Comment by im-coder-lg** [comment]
https://user-images.githubusercontent.com/76845820/155736592-5709df4a-332a-462d-9160-60eecf2486c5.mp4

The first part is just opening LXTerminal, `cd`ing to the git folder, pulling changes, checking out and running the code. THen commenting the blur code out(it fails yet) and starting. At first it seems to work, but look at the theme switching.

**Comment by im-coder-lg** [comment]
I'm almost done with this! Just got some interest to fix this, I'll commit in another branch so that you can add/modify it as you want.

**Comment by im-coder-lg** [comment]
Look at #96!

**Comment by not-nef** [comment]
Added a github link thing to close the issue when this pr gets merged.

**Comment by sumeshir26** [comment]
@not-nef can you review?

**Pull request merged**